lib/tasks/case_contact_types_reminder.rb

```diff
@@ -0,0 +1,96 @@
+class CaseContactTypesReminder
+  include Rails.application.routes.url_helpers
+
+  NEW_CASE_CONTACT_PAGE_PATH = "https://casavolunteertracking.org/case_contacts/new"
```

Comment:
do not use static urls, they will break in non-prod environments

Reply:
Is there a way to access base URL for the current environment the app is running in?

Reply:
Used environment variables in encrypted yml files to have base url for each environment.